src/datagrid/DataGrid.stories.tsx

```diff
@@ -438,4 +440,70 @@ storiesOf("DataGrid", module)
                 footer={hideShowColFooter}
             />
         </div>
-    ));
+    ))
+    .add("Grid Demo with support to open/close details pane", () => {
```

Comment:
Can we change story name something like,
Grid with open/close details pane on link click ??

Reply:
Done